tasks/integration/_linux-macos-shared.yml

```diff
@@ -9,16 +23,18 @@
       - "{{ item.key }}"
   become: yes
   become_user: "{{ integration_command_user }}"
-  loop: "{{ datadog_integration|dict2items }}"
-  when: item.value.action == "remove"
+  when: 
+    - item.value.action == "remove"
+    - integration_version.stdout | length > 0
```

Comment:
I think it's probably more usual to check the return code (`.rc`) rather than output length.

Reply:
Yes you're right, thanks! I will change this.